### PR TITLE
Bump to VirtualBox 6.1.8

### PR DESCRIPTION
**Pull request by jimklimov**
New vbox-6.1.8 was released, changelog indicates that a couple of recent regressions were addressed that bit me (I hope to check today if they really were), so it may be worthwhile to bump the one in OI if not yet. Namely, for guest additions: it should (still/again) work with clipboard, and ("newly" via mew mechanisms after old hacks were dropped) support resizing; for hosts: should not block into slow teletype mode when the TCP listener of a serial port is not actually listened.

Unfortunately, I fail to consistently build, install and package the recipe and not sure why. So feel free to pick up the updated version/hash data and make a usable package :) I would not have time to proceed on this in the nearby weeks, unless it were a quick win :(

### CODE REVIEW

File: components/sysutils/virtualbox/Makefile [hunk 1]
```diff
@@ -152,9 +156,10 @@ CONFIGURE_OPTIONS+= --nofatal
 CONFIGURE_OPTIONS+= --enable-webservice
 CONFIGURE_OPTIONS+= --enable-vnc
 
-CONFIGURE_OPTIONS.32 += --only-additions
+CONFIGURE_OPTIONS.32 = --only-additions
 CONFIGURE_OPTIONS.32 += --target-arch=x86
-CONFIGURE_OPTIONS.64 += --enable-qt5
+
```
Comment (pyhalov) [inline components/sysutils/virtualbox/Makefile]: Please, remove empty line.

Comment (alarcher) [reply components/sysutils/virtualbox/Makefile]: Fixed and added a patch for gcc-10.

**Comment by alarcher** [comment]
> New vbox-6.1.8 was released, changelog indicates that a couple of recent regressions were addressed that bit me (I hope to check today if they really were), so it may be worthwhile to bump the one in OI if not yet. Namely, for guest additions: it should (still/again) work with clipboard, and ("newly" via mew mechanisms after old hacks were dropped) support resizing; for hosts: should not block into slow teletype mode when the TCP listener of a serial port is not actually listened.
> 
> Unfortunately, I fail to consistently build, install and package the recipe and not sure why. So feel free to pick up the updated version/hash data and make a usable package :) I would not have time to proceed on this in the nearby weeks, unless it were a quick win :(

I noticed that the parallel build can exhaust 32GB of RAM and fail.

**Review by alarcher — APPROVED** [review]
Add gcc-10 patch.

**Pull request merged**